### ISSUE TITLE
Merge output parts together

### ISSUE THREAD

**Issue by elmopl** [elmopl/ktba#4]
I am not sure how hard this will be though.

**Comment by elmopl** [comment]
That's the one thing I don't think should be part of this addon.

The reasoning behind it is that Blender is not capable of stitching videos.
Also encoding configuration in Blender is also rather lacking (which by no means is a complaint on Blender, it is simply not the purpose of this tool).

If you are editing video you want to do your final encoding with some external tool, which most likely will be able to handle stitching too.

Now, having separate output files and trying to merge them also poses problems with audio.
If you have audio and you simply stitch videos (by whatever means) audio will most likely have noticeable issues at stitch points.

To sum up, the workflow I'd recommend:

 - use Blender to generate video parts without audio and with loseless video setting
 - bake audio to separate file
 - stitch all parts and replace audio (i.e. using [Avidemux](http://avidemux.sourceforge.net/))
 - encode final edit (i.e. using [Handbrake](https://handbrake.fr/))
